static/index.html

```diff
@@ -57,6 +57,31 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. You can also extend these with
             support for your own builds.</p>
+            <h2 id="overview"><a href="#overview">Overview</a></h2>
+            <ul>
+                <li>
+                    GrapheneOS does <em>not</em> support Google play services. Some apps may not work, or only have partial functionality.
+                </li>
+                <li>
+                    GrapheneOS does <em>not</em> support rooting. This goes against its design goals and will <b>never</b> be supported.
+                </li>
+                <li>
+                    If you are looking to buy a phone to run GrapheneOS, the Pixel 3(xl) and 3a(xl) are the best options.
+                </li>
+                <li>
+                    GrapheneOS currently has no plans to support the Pixel 4 or Pixel 4XL
+                </li>
+                <li>
+                    This OS is 'barebones' and does not come with many preinstalled apps ontop of AOSP.
+                </li>
+                <li>
+                    You will most likely have problems running GrapheneOS on a carrier locked Pixel (eg. Verizon Pixels)
```

Comment:
This isn't clear and also doesn't belong on the main page. It's not an attribute of the OS.